doc/source/programs/gdal_raster_pixelinfo.rst

```diff
@@ -0,0 +1,238 @@
+.. _gdal_raster_pixelinfo:
+
+================================================================================
+``gdal raster pixelinfo``
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Return information on a pixel of a raster dataset
+
+.. Index:: gdal raster pixelinfo
+
+Synopsis
+--------
+
+.. program-output:: gdal raster pixelinfo --help-doc
+
+Description
+-----------
+
+:program:`gdal raster pixelinfo` provide a mechanism to query information about
+a pixel given its location in one of a variety of coordinate systems.
+
+It supports outputting either as GeoJSON or CSV.
+
+The following items will be reported (when known):
+
+- Input coordinates
+- Input coordinates converted to column, line
```

Comment:
I would consider making some of these opt-in to get a smaller output dataset.

Reply:
just outputting the floating point column, line coordinates instead of the _int and _double version